lib/helpers.js

```diff
@@ -117,6 +119,23 @@ function getToken(userInfo) {
       }
     }
     return accessToken;
+  } else if (cmd && cmd === 'aws') {
+    const args = _.get(userInfo, 'user.exec.args');
+    if (args) {
+      cmd = `${cmd} ${args.join(' ')}`;
+    }
+    const profile = _.find(userInfo.user.exec.env, e => e.name === 'AWS_PROFILE');
```

Comment:
if `userInfo.user.exec` is undefined this will crash, you can try:

```
const env = _.get(userInfo, 'user.exec.env', []);
const profile = ...
```